setup.py

```diff
@@ -1,10 +1,14 @@
 from setuptools import setup, find_packages
 
 setup(name='dpo',
-      version='0.0.2',
+      version='0.1.0',
```

Comment:
```suggestion
      version='0.1.1',
```

bump version number up for next release